Logos/atlassian.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Svg file should not contain xmlns:xlink.Please refer the below mention checklist for logo:

Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
xlink:href is not there and inline is used instead
title tag is not used

Reply:
Id should be of GUID. Could you please update the id in guid type.

Please refer https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md for logo guidelines.

Thanks,
Aditi Maheshwari

Reply:
> Svg file should not contain xmlns:xlink.Please refer the below mention checklist for logo:
> 
> Ensure raw file of logo does not have any of the following:
> cls and style formats
> embedded png formats
> xmlns:xlink
> data-name
> xlink:href is not there and inline is used instead
> title tag is not used

@averbn  : We have resolved the issues found in the svg file. Please have a look, you can use this file.

[atlassian.zip](https://github.com/Azure/Azure-Sentinel/files/5963456/atlassian.zip)

Reply:
@averbn  could you please check the comments and resolved them.

Thanks,
Aditi Maheshwari

Reply:
Changed icon according to your requirements. Pls check